### DISCUSSION TITLE
colour mapping

### DISCUSSION THREAD

**Discussion by PINACI-Leather** [arkypita/LaserGRBL#1371] [Ideas]
Will there ever be colour mapping in LaserGRBL ? It would be nice to be able to set different power settings to different coloured lines in your designs. For example I would like to engrave images on my leather designed using my normal engraving speed and power but also like to draw the outline shape and punch holes into the leather at a lower power so they only just show up enough for me to cut out by hand.

**Comment by arkypita** [comment]
Hi, thank for suggestion
It is already in #64 Development Roadmap but it is not so simple to implement as you may think.
